streamwebs_frontend/streamwebs/models.py

```diff
@@ -41,7 +41,7 @@ class Site(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     modified = models.DateTimeField(auto_now=True)
 
-    objects = SiteManager()
+    test_objects = SiteManager()
```

Comment:
Was there a reason for renaming this? Looks like you had to edit all the other datasheets to fix  the tests.